src/TestPlan.lua

```diff
@@ -196,7 +196,9 @@ function TestNode:expand()
 	end
 	setfenv(self.callback, callbackEnv)
 
-	local success, result = xpcall(self.callback, debug.traceback)
+	local success, result = xpcall(self.callback, function(message)
+		return debug.traceback(tostring(message), 2)
```

Comment:
Nitpick: Might be good to later move this debug.traceback snippet into its own modules for the sake of making sure we keep it DRY, esp since we're passing in "magic" scope

Reply:
There's probably several such utility functions that could be pulled out.